products/ceqr_survey/models/_sources.yml

```diff
@@ -89,12 +89,37 @@ sources:
         columns:
           - name: permit_id
             tests:
-              - unique
+              - unique # TODO: need to confirm this assumption
```

Comment:
Confirmed?

Reply:
agree we can consider our testing of it as confirmation